components/slack/actions/create-reminder/create-reminder.js

```diff
@@ -0,0 +1,48 @@
+const slack = require("../../slack.app.js");
+const { WebClient } = require("@slack/web-api");
```

Comment:
Small nit, but please add a line between the `require`s and the module.

Reply:
Got it